mongodbatlas/projects.go

```diff
@@ -208,6 +209,33 @@ func (s *ProjectsServiceOp) Create(ctx context.Context, createRequest *Project,
 	return root, resp, err
 }
 
+// Update updates a project.
+//
+// https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Projects/operation/updateProject
+func (s *ProjectsServiceOp) Update(ctx context.Context, projectID string, createRequest *Project) (*Project, *Response, error) {
```

Comment:
```suggestion
func (s *ProjectsServiceOp) Update(ctx context.Context, projectID string, updateRequest *Project) (*Project, *Response, error) {
```

Reply:
also only name is valid, not sure if you should use a different struct that doesn't include all the fields

Reply:
ops thanks. I thought about the new struct as well but I left `Project` in case the endpoint will support additional fields. However, It seems quite unlikely since the endpoint is called "update project name"... I will create a new struct if you don't have a strong opinion

Reply:
not too opinionated if doing or not

Reply:
Updated